library/Exceptions/ValidationException.php

```diff
@@ -173,7 +172,7 @@ protected function buildTemplate()
     {
         $templateKey = $this->chooseTemplate();
 
-        return static::$defaultTemplates[$this->mode][$templateKey];
+        return @ Validator::$messages[basename(get_called_class())][$this->mode][$templateKey];
```

Comment:
This is also for the changes above: don't use shut up operator and make the verification instead, you can throw an exception, for example.